setup/k8s/celery.yaml

```diff
@@ -34,11 +34,6 @@ spec:
       labels:
         pod: celery-worker
     spec:
-      # The Celery worker and beat processes run as a non-root user (UID 1000)
-      securityContext:
-        runAsUser: 1000
```

Comment:
Issues with running Celery as non-root user were successfully resolved?

Reply:
The team agreed to postpone the work of making sure all the processes are running as non-root.